docs/a-walk-through-vhs.md

```diff
@@ -0,0 +1,250 @@
+# A Walk Through VHS
+
+Today we're going to take a walk through VHS. We'll start from a manifest URL and end with video playback.
+
+The purpose of this walk is not to see every piece of code, or define every module. Instead it's about seeing the most important parts of VHS. The goal is to make VHS more approachable.
+
+Lets start with a video tag:
+
+```html
+<video>
+  <source src="http://example.com/manifest.m3u8" type="application/x-mpegURL">
+</video>
+```
+
+The source, `manifest.m3u8`, is an HLS manifest. You can tell from the `.m3u8` extension and the `type`.
+
+Safari (and a few other browsers) will play that video natively, because Safari supports HLS content. However, other browsers don't support native playback of HLS and will fail to play the video.
+
+VHS provides the ability to play HLS (and DASH) content in browsers that don't support native HLS (and DASH) playback.
+
+Since VHS is a part of Video.js, let's set up a Video.js player for the `<video>`:
+
+```html
+<link href="//vjs.zencdn.net/7.10.2/video-js.min.css" rel="stylesheet">
+<script src="//vjs.zencdn.net/7.10.2/video.min.js"></script>
+
+<video-js id="myPlayer" class="video-js" data-setup='{}'>
```

Comment:
Wondering if we should remove `data-setup` here. I'm not sure we want to encourage its use as a feature.

Also, don't need `class="video-js"` if using the `<video-js>` element.